src/pages/Documents.jsx

```diff
@@ -0,0 +1,144 @@
+import React, { useContext, useEffect, useState } from 'react';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import { useLocation } from 'react-router-dom';
+import { useTheme } from '@emotion/react';
+import { useMediaQuery } from '@mui/material';
+import { ConfirmationModal, SetAclPermissionsModal, UploadDocumentModal } from '@components/Modals';
+import { useNotification, useSession } from '@hooks';
+import { DocumentListContext } from '@contexts';
+import { truncateText } from '@utils';
+import { DocumentTable } from '@components/Documents';
+import { Container } from '@mui/system';
+
+const Documents = () => {
```

Comment:
No JSDocs?

Reply:
Given it's a page and not exactly a component, didn't think it'll be necessary to write one, but I can include a short documentation